README.md

```diff
@@ -1,43 +1,25 @@
 # brainverse
-## For Developers
 
-## Requirements
-- [Node.js](https://nodejs.org/en/download/) (which comes with npm) installed on your computer
+## To use the App in MacOS
+- Download the App (DMG file) from the [Dropbox link](https://www.dropbox.com/s/41fx3ufnrej1d02/BrainVerse-0.0.1.dmg?dl=0)
+- Double click on the dmg file
+- Drag the app icon to the Application folder, it will start installing
+- To start the app, double click on the app icon in the Application folder
+- All the files created are stored outside the Brainverse App folder under /Users/username/Documents/uploads.
 
-- Git
+### Linux
 
```

Comment:
empty - shouldn't the apps be built on travis?

Reply:
there are still some errors on travis. since travis is building the dmg, zip for linux and windows builds, shouldn't they be pushed to a location people can download from?

Reply:
yes, ideally travis after build should be able to publish the app for all three OSes. I have not figured it out yet. When I was working on Linux configuration, I was getting error so .travis.yml only contains instruction for macOS. I will be updating the built configuration soon.